topic04-git-introduction/book-git-tutorial/01.Basics.md

```diff
@@ -0,0 +1,30 @@
+# **_Git guide to commands_**
+
+This manual contains necessary commands for git when working on your own, contributing to other projects and working as a team.
+
+###### What is git about?
+
+Git is a network that allows you to manage your code easily while working as a team or contributing to other repos, It allows you to merge code without having to copy and paste it from another file or files. Git also is able to host your website but it does not give you a proper full domain. 

```

Comment:
[Change Required] rather than `network` I would say that its a `version control system`

Reply:
[Change Required] `... contributing to other repos,` I think that's supposed to be a `.` not a `,`

Reply:
[Change Required] I would also add here that you can host your website `using github pages` and add a link to https://pages.github.com/ for further information.